tt_eager/tensor/borrowed_buffer.hpp

```diff
@@ -31,6 +31,7 @@ struct Buffer {
     inline const T* end() const noexcept { return this->data_ptr_ + this->size(); }
 
     inline void* data() noexcept { return static_cast<void*>(this->data_ptr_); }
+    inline const void* get_ptr() const noexcept { return static_cast<void*>(this->data_ptr_); }
```

Comment:
Why not use `data` method that already exists?

Reply:
`get_ptr()` removed for `borrowed_buffer` class